src/polymer/expression-scanner.ts

```diff
@@ -128,6 +138,67 @@ export class DatabindingExpression {
     return correctSourceRange(
         databindingRelativeSourceRange, this.locationOffset);
   }
+
+  private _extract() {
+    if (this._expressionAst.body.length !== 1) {
+      this.warnings.push(this._validationWarning(
+          'Expected exactly one expression', this._expressionAst));
+      return;
+    }
+    const expressionStatement = this._expressionAst.body[0]!;
+    if (expressionStatement.type !== 'ExpressionStatement') {
+      this.warnings.push(this._validationWarning(
+          'ExpressionStatement expected', expressionStatement));
```

Comment:
Fair. I changed to a two sentence warning style in a couple places here. The first gives the problem in plain language, the second gives more technical detail that will be helpful to experts (including us, when people report issues).